src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashPostBuildCommentAction.java

```diff
@@ -3,19 +3,10 @@
 import hudson.model.InvisibleAction;
 
 public class StashPostBuildCommentAction extends InvisibleAction {
```

Comment:
Maybe we should rename the class since it's not longer about build comments? (Using that rename class migration linked above)

Reply:
The only purpose of that class now is to be a compatibility data consumer/destroyer of its former self, i.e. `StashPostBuildCommentAction`. I cannot think of a better name for it.

Reply:
OK, nevermind then